planning/autoware_costmap_generator/src/costmap_generator.cpp

```diff
@@ -374,12 +365,11 @@ grid_map::Matrix CostmapGenerator::generatePointsCostmap(
   return points_costmap;
 }
 
-autoware_perception_msgs::msg::PredictedObjects::ConstSharedPtr transformObjects(
-  const tf2_ros::Buffer & tf_buffer,
-  const autoware_perception_msgs::msg::PredictedObjects::ConstSharedPtr in_objects,
+PredictedObjects::ConstSharedPtr transformObjects(
+  const tf2_ros::Buffer & tf_buffer, const PredictedObjects::ConstSharedPtr in_objects,
   const std::string & target_frame_id, const std::string & src_frame_id)
 {
-  auto objects = new autoware_perception_msgs::msg::PredictedObjects();
+  auto objects = new PredictedObjects();
```

Comment:
```suggestion
   auto objects = std::make_shared<autoware_perception_msgs::msg::PredictedObjects>();
```

nits: better not to use raw pointer considering like exception. same to ohter parts too.